ResearchKit/Common/ORKResult.m

```diff
@@ -1670,6 +1671,65 @@ - (ORKStepResult *)stepResultForStepIdentifier:(NSString *)stepIdentifier {
 
 @end
 
+@implementation ORKLocationQuestionResult
+
+- (void)encodeWithCoder:(NSCoder *)aCoder {
+    [super encodeWithCoder:aCoder];
+    ORK_ENCODE_OBJ(aCoder, locationAnswer);
+}
+
+- (instancetype)initWithCoder:(NSCoder *)aDecoder {
+    self = [super initWithCoder:aDecoder];
+    if (self) {
+        ORK_DECODE_OBJ_CLASS(aDecoder, locationAnswer, NSValue);
+    }
+    return self;
+}
+
++ (BOOL)supportsSecureCoding {
+    return YES;
+}
+
+- (BOOL)isEqual:(id)object {
+    BOOL isParentSame = [super isEqual:object];
+    
+    __typeof(self) castObject = object;
+    return (isParentSame &&
+            //ORKEqualObjects(_locationAnswer, castObject.locationAnswer)) ;
+            _locationAnswer.MKCoordinateValue.latitude == castObject.locationAnswer.MKCoordinateValue.latitude &&
+            _locationAnswer.MKCoordinateValue.longitude == castObject.locationAnswer.MKCoordinateValue.longitude);
+}
+
+- (NSUInteger)hash {
+    return [super hash];
+}
+
+- (instancetype)copyWithZone:(NSZone *)zone {
+    ORKLocationQuestionResult *result = [super copyWithZone:zone];
+    result->_locationAnswer = [self.locationAnswer copyWithZone:zone];
+    return result;
+}
+
++ (Class)answerClass {
+    return [NSValue class];
+}
+
+- (void)setAnswer:(id)answer {
+    if ([answer isKindOfClass:[NSArray class]]) {
+        // Because ORKLocationAnswerFormat has ORKChoiceAnswerFormat as its implied format.
```

Comment:
Is this true? ` ORKLocationAnswerFormat has ORKChoiceAnswerFormat as its implied format.`